### ISSUE TITLE
Look over `Java with maven´-action

### ISSUE THREAD

**Issue by HansenChristoffer** [HansenChristoffer/DeviceConnectivityPlatform#10]

**Comment by HansenChristoffer** [comment]
Blocked by #3

**Comment by HansenChristoffer** [comment]
No longer blocked by #3. Can be worked upon.

**Comment by HansenChristoffer** [comment]
Should TM be solved. Might open a new issue ticket if it still don't work for whatever reason in the future.
